lib/components/search_user/interactor/search_user_bloc.dart

```diff
@@ -39,8 +39,8 @@ class SearchUserBloc extends Bloc<SearchUserEvent, SearchUserState> {
 
       if (event.searchQuery.length > _minTextLengthBeforeValidSearch) {
         yield state.copyWith(pageState: PageState.loading);
-        final result = await SearchForMemberUseCase().run(event.searchQuery);
-        yield SearchUserStateMapper().mapResultToState(state, result, state.noShowUsers);
```

Comment:
we could filter the telos users by seeds users in the use case, but generally we do all logic in the mapper so I think it should be done there, that's why we have a mapper.